### PR TITLE
Propose adding a new subcat to the CNAI group and adding Chainguard Images to it

### PR DESCRIPTION
**Pull request by MordodeMaru**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other [guidelines](https://github.com/cncf/landscape#new-entries)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?

### CODE REVIEW

**Comment by tegioz** [comment]
@castrojo please note that the error you see in the preview is due to an error introduced by the changes suggested by this PR. So this often will be an alert that something is wrong and needs to be fixed 🙂 This is something the validate check will report as well as soon as it's approved (maybe we could consider authorizing this check to run without approval so that users can be made aware of the errors and fix them).

Screenshot 2024-05-15 at 16 37 30

<br/>

Screenshot 2024-05-15 at 16 33 23

**Review by MordodeMaru — COMMENTED** [review]
I think I have amended the error in line 18395

**Review by cynthia-sg — CHANGES_REQUESTED** [review]

File: landscape.yml
Comment (MordodeMaru) [reply landscape.yml:18396]: Sure. How do I request a new subcategory?

File: landscape.yml
Comment (MordodeMaru) [reply landscape.yml:18412]: Indeed

File: landscape.yml
Comment (MordodeMaru) [reply landscape.yml:18417]: Ok.

File: landscape.yml
Comment (cynthia-sg) [reply landscape.yml:18406]: Delete the line, please.

File: landscape.yml
Comment (MordodeMaru) [reply landscape.yml:18410]: The community meets in the #apko channel of that Slack community.

File: landscape.yml
Comment (MordodeMaru) [reply landscape.yml:18396]: I don't think we will add Chainguard Images if it's not to the right category. We need to solve that first.

Comment (cynthia-sg) [reply landscape.yml:18396]: You could bring it up in the landscapers channel in Slack, maybe in this [thread](https://cloud-native.slack.com/archives/CM09QERF1/p1714450672828699) where the CNAI landscape group has been discussed recently.

Comment (MordodeMaru) [reply landscape.yml:18396]: I will. Thanks!

File: landscape.yml
Comment (cynthia-sg) [reply landscape.yml:18410]: Ok, you can use `[chat_channel](chat_channel)` field

**Comment by MordodeMaru** [comment]
Closing this PR until I get some clarity on category creation.

**Comment by MordodeMaru** [comment]
Closing this PR until I get some clarity on category creation.

**Pull request closed without merging**
